src/lib.rs

```diff
@@ -83,7 +83,7 @@ pub use crate::term::{
     user_attended, user_attended_stderr, Term, TermFamily, TermFeatures, TermTarget,
 };
 pub use crate::utils::{
-    colors_enabled, colors_enabled_stderr, measure_text_width, pad_str, pad_str_with,
+    char_width, colors_enabled, colors_enabled_stderr, measure_text_width, pad_str, pad_str_with,
```

Comment:
I'd prefer not to expose this.

Reply:
Because I want to use it in https://github.com/console-rs/dialoguer/pull/321